renderer/reducers/activity.js

```diff
@@ -141,12 +140,28 @@ const transactionsSelector = state => transactionsSelectors.transactionsSelector
 const transactionsSendingSelector = state => state.transaction.transactionsSending
 const modalItemTypeSelector = state => state.activity.modal.itemType
 const modalItemIdSelector = state => state.activity.modal.itemId
+const nodesSelector = state => state.network.nodes
 
 const invoiceExpired = invoice => {
   const expiresAt = parseInt(invoice.creation_date, 10) + parseInt(invoice.expiry, 10)
   return expiresAt < Math.round(new Date() / 1000)
 }
 
+/**
+ * Augment payments with additional data.
+ */
+const decoratedPaymentsSelector = createSelector(
+  paymentsSelector,
+  nodesSelector,
+  (payments, nodes) =>
+    payments.map(payment => {
+      const destPubKey = payment.path[payment.path.length - 1]
```

Comment:
Is `payment.path` always set?

Reply:
It should be, but I'ave added some safeguards around this just incase.